cdap-ui/app/cdap/api/pipeline.js

```diff
@@ -27,6 +27,8 @@ let programPath = `${basepath}/:programType/:programName`;
 export const MyPipelineApi = {
   publish: apiCreator(dataSrc, 'PUT', 'REQUEST', basepath),
 
+  getSchedule: apiCreator(dataSrc, 'GET', 'REQUEST', `${schedulePath}`),
+  updateSchedule: apiCreator(dataSrc, 'POST', 'REQUEST', `${schedulePath}/update`),
```

Comment:
nit: Both of these are already in `MyScheduleApi`. Is there a reason why we have to add them here?